docs/usage/gpu.md

```diff
@@ -0,0 +1,212 @@
+---
+title: GPU Enabled Cluster
```

Comment:
I guess https://github.com/gardener/gpu-installer is much better place for this guide.